packages/pds-ember/addon/components/pds/app/body/index.hbs

```diff
@@ -0,0 +1,9 @@
+<main
+  class="pds-app__body"
+  id="content"
```

Comment:
This allows the "skip" link to wire up to body content, automatically.

Reply:
`content` is a bit generic - since it's an `id` I imagine that there could be collision with this - should we document it? or maybe prefix it? `pds-content`?

Reply:
It's possible, but I'd rather address that issue when/if it actually becomes a problem.